cfgov/unprocessed/css/pages/ask.less

```diff
@@ -105,6 +104,99 @@
 /* Answer page */
 
 .ask-cfpb-page__answer {
+    .respond-to-min( @bp-lg-min, {
+        .content_sidebar {
+            padding-top:0;
+        }
+    });
```

Comment:
TRIVIAL: mix of spaces and not spaces between parens and colons here and throughout.

Reply:
Should be fixed with latest commits.